recipes/discorpy/meta.yaml

```
@@ -0,0 +1,42 @@
+package:
+  name: discorpy
+  version: 1.4
+
+source:
+  url: https://github.com/DiamondLightSource/discorpy/archive/refs/tags/v1.4.tar.gz
```

Then use the template variables here:
```suggestion
  url: https://github.com/DiamondLightSource/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
```